cmd/dartboard/subcommands/load.go

```diff
@@ -141,7 +144,7 @@ func loadProjects(r *dart.Dart, kubeconfig string, clusterName string, clusterDa
 
 	log.Printf("Load resources on cluster %q (#Projects: %s)\n", clusterName, projectCount)
 
-	if err := kubectl.K6run(kubeconfig, "k6/create_projects.js", envVars, tags, true, clusterAdd.Local.HTTPSURL, false); err != nil {
+	if err := kubectl.K6run(kubeconfig, "create_projects.js", envVars, tags, true, clusterAdd.Local.HTTPSURL, false); err != nil {
```

Comment:
I'm a little bit worried with this and the other two commands not specifying the file directly and instead walking the tree. What happens if multiple files have this same name, but are in different directories?

Reply:
Correction: The tree-walking logic is needed due to the refactor to use a nested directory structure under the `k6/` dir. So we need to maintain the relative file paths when the configmap gets mounted to the tester cluster, otherwise the relative filepath imports within the k6 scripts will not be valid.